### PR TITLE
[MODGOBI-189] - Release v2.6.0

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+[Full Changelog] (https://github.com/folio-org/mod-gobi/compare/v2.5.0...v2.6.0)
+
+### Stories
+* [MODGOBI-186](https://issues.folio.org/browse/MODGOBI-186) Remove 'Lookup mock' translator from gobi configuration
+* [MODGOBI-178](https://issues.folio.org/browse/MODGOBI-178) Logging improvement - Configuration
+* [MODGOBI-146](https://issues.folio.org/browse/MODGOBI-146) Logging improvement
+
+### Bug fixes
+* [MODGOBI-188](https://issues.folio.org/browse/MODGOBI-188) Organization lookup hardcoded
+* [MODGOBI-183](https://issues.folio.org/browse/MODGOBI-183) Mapper.multiply ignores a single value
```
Comment (SerhiiNosko) [inline NEWS.md:15]: I think these 2 issues included already in 2.5.1 release so we don't need to specify them here

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
```
Comment (SerhiiNosko) [inline NEWS.md:16]: <mod-configuration-client.version>5.9.1-SNAPSHOT</mod-configuration-client.version>
We need to setup the latest release version in pom.xml

File: NEWS.md (hunk 1 above)
Comment (Dmitriy-Butramyou) [reply NEWS.md:15]: We don't have those bugs in v2.5.1 release notes

Comment (SerhiiNosko) [reply NEWS.md:15]: ok got it, MODGOBI-188 Organization lookup hardcoded - its our recent bug, 
and for MODGOBI-183 - the fix version is 2.6.0

Thanks

File: NEWS.md (hunk 2 above)
Comment (Dmitriy-Butramyou) [reply NEWS.md:16]: Version updated to 5.9.1

**Review by SerhiiNosko — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**